testing/integration/selected_parent_chain_test.go

```diff
@@ -15,7 +16,7 @@ func TestVirtualSelectedParentChain(t *testing.T) {
 
 	// Register to virtual selected parent chain changes
 	onVirtualSelectedParentChainChangedChan := make(chan *appmessage.VirtualSelectedParentChainChangedNotificationMessage)
-	err := kaspad1.rpcClient.RegisterForVirtualSelectedParentChainChangedNotifications(
```

Comment:
nit: Edit this integration test to check if it works propertly

Reply:
Hmmmm... I looked into that, it would be quite complex to modify this test to include checking acceptedTransactionIDs because this tests checks something completely different.

So I did the only responsible thing possible, and created an issue: https://github.com/kaspanet/kaspad/issues/2040